README.md

```diff
@@ -8,6 +8,17 @@ This repository contains a series of tools, templates, tips and tricks to make y
 * [DocLinkChecker](./src/DocLinkChecker): validate links in documents and check for orphaned attachments in the `.attachments` folder. The tool indicates whether there are errors or warnings, so it can be used in a CI pipeline. It can also clean up orphaned attachments automatically. And it can validate table syntax.
 * [DocLanguageTranslator](./src/DocLanguageTranslator): allows to generate and translate automatically missing files or identify missing files in multi language pattern directories.
 
+## Install
+
+The tools can be installed by dowloading the zip-file of a [release](https://github.com/Ellerbach/docfx-companion-tools/releases) or use [Chocolatey](https://chocolatey.org/install) like this:
+
+```shell
+choco install docfx-companion-tools
+```
+
```

Comment:
Good, maybe how to run them here as well

Reply:
Added more details how to run after install in README.